### DISCUSSION TITLE
souldance/

### DISCUSSION THREAD

**Discussion by giscus** [OweQian/oweqian.github.io#10] [General]
# souldance/

🚇 上，🏃🏻 ‍时，搬 🧱 中……戴起耳机，就是你一个人的舞台！在这里，希望你能找到一首 BGM，来撑起你独一无二的内心戏

https://oweqian.xyz/souldance/

**Comment by EricZZZ** [comment]
搞得我都想整个Hi-Fi耳机了😄
